doc/api/next_api_changes/2018-09-18-AL-removals.rst

```diff
@@ -0,0 +1,47 @@
+API removals
+````````````
+
+The following deprecated APIs were removed:
+
+Classes and methods
+-------------------
+- ``Verbose`` (replaced by python logging library)
+- ``artist.Artist.hitlist`` (no replacement)
+- ``artist.Artist.is_figure_set`` (use ``artist.figure is not None`` instead)
+- ``axis.Axis.unit_data`` (use ``axis.Axis.units`` instead)
+- ``backend_bases.FigureCanvasBase.onRemove`` (no replacement)
+  ``backend_bases.FigureManagerBase.show_popup`` (this never did anything)
+- ``backend_wx.SubplotToolWx`` (no replacement)
+- ``backend_wx.Toolbar`` (use ``backend_wx.NavigationToolbar2Wx`` instead)
+- ``cbook.align_iterators`` (no replacment)
```

Comment:
Perhaps say "use pandas instead" or something?

Reply:
If you have a specific replacement in mind, please push it (or I can do it), but otherwise that seems a bit vague (at least I don't know what replacement you're thinking about).